src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -112,7 +112,6 @@
     <VSIXSourceItem Include="$(OutputPath)Microsoft.Extensions.FileProviders.Physical.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServer.Protocol.dll" />
     <VSIXSourceItem Include="$(OutputPath)Microsoft.VisualStudio.LanguageServer.Protocol.Extensions.dll" />
-    <VSIxSourceItem Include="$(OutputPath)System.IO.Pipelines.dll" />
```

Comment:
Things may start following over without this now due to some upstream changes. Just fyi in case CI fails.